osu.Game/Screens/Utility/LatencyArea.cs

```diff
@@ -0,0 +1,239 @@
+                    Text = "Feels better",
+                    Y = 20,
+                    Width = 0.8f,
+                    Anchor = Anchor.TopCentre,
+                    Origin = Anchor.TopCentre,
+                    Action = () => ReportUserBest?.Invoke(),
+                },
+                new Container
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    Children = new Drawable[]
+                    {
+                        new LatencyMovableBox(IsActiveArea)
+                        {
+                            RelativeSizeAxes = Axes.Both,
+                        },
+                        new LatencyCursorContainer(IsActiveArea)
+                        {
+                            RelativeSizeAxes = Axes.Both,
+                        },
+                    }
+                },
+            };
+
+            IsActiveArea.BindValueChanged(active =>
+            {
+                background.FadeColour(active.NewValue ? overlayColourProvider.Background4 : overlayColourProvider.Background6, 200, Easing.OutQuint);
+            }, true);
+        }
+
+        protected override bool OnMouseMove(MouseMoveEvent e)
+        {
+            IsActiveArea.Value = true;
+            return base.OnMouseMove(e);
+        }
+
+        private double lastFrameTime;
+
+        public override bool UpdateSubTree()
+        {
+            double elapsed = Clock.CurrentTime - lastFrameTime;
+            if (TargetFrameRate > 0 && elapsed < 1000.0 / TargetFrameRate)
+                return false;
+
+            lastFrameTime = Clock.CurrentTime;
+
+            return base.UpdateSubTree();
+        }
+
+        public class LatencyMovableBox : CompositeDrawable
+        {
+            private Box box = null!;
+            private InputManager inputManager = null!;
+
+            private readonly BindableBool isActive;
+
+            [Resolved]
+            private OverlayColourProvider overlayColourProvider { get; set; } = null!;
+
+            public LatencyMovableBox(BindableBool isActive)
+            {
+                this.isActive = isActive;
+            }
+
+            protected override void LoadComplete()
+            {
+                base.LoadComplete();
+
+                inputManager = GetContainingInputManager();
+
+                InternalChild = box = new Box
+                {
+                    Size = new Vector2(40),
+                    RelativePositionAxes = Axes.Both,
+                    Position = new Vector2(0.5f),
+                    Origin = Anchor.Centre,
+                    Colour = overlayColourProvider.Colour1,
+                };
+            }
+
+            protected override bool OnHover(HoverEvent e) => false;
+
+            private double? lastFrameTime;
+
+            protected override void Update()
+            {
+                base.Update();
+
+                if (!isActive.Value)
+                {
+                    lastFrameTime = null;
```

Comment:
To fix the inactive box color being stuck when changing active state while pressing keys.
```suggestion
                    lastFrameTime = null;
                    box.Colour = overlayColourProvider.Colour1;
```